src/main/java/Duke.java

```diff
@@ -1,10 +1,236 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Duke {
+
+    private static Scanner in = new Scanner(System.in);
+    static final int MAX_SIZE = 100;
+    private static String content = "";
+
+    private static String lineInput = "";
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+    private static int tasksCount = 0;
+
+    public static void addTask(Task t){
+        if (t.getDescription().length() > 0) {
+            System.out.println("Got it. I've added this task:");
+            taskList.add(t);
+            tasksCount++;
+            System.out.println(t);
+            System.out.println("Now you have " + tasksCount + " tasks on the list.");
+        }
+    }
+
+    public static void addTask(Task t, boolean b){
+        if (t.getDescription().length() > 0) {
+            taskList.add(t);
+            tasksCount++;
+        }
+    }
+
+
+    private static void listTasks(){
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 1; i <= tasksCount; i++) {
+            Task currTask = taskList.get(i - 1);
+            System.out.println(i + ". " + currTask);
+        }
+    }
+
+    private static void markTaskDone(int index) {
+        taskList.get(index).markDone();
+        Task currTask = taskList.get(index);
+        System.out.println((index + 1) + ". " + currTask);
+    }
+
+    private static void deleteTask(int index) {
+        Task currTask = taskList.get(index);
+        System.out.println("Noted. I've removed this task:");
+        taskList.remove(index);
+        System.out.println(currTask);
+        tasksCount--;
+        System.out.println("Now you have " + tasksCount + " tasks on the list.");
+    }
+
+    private static void printBorder(){
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    private static void printFileContents(String filePath) throws FileNotFoundException {
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        while (s.hasNext()) {
+            String currentLine = s.nextLine();
+            String lineType = currentLine.split("\\|", 3)[0];
+            String lineIsDone = currentLine.split("\\|", 3)[1];
+            String lineAction = currentLine.split("\\|", 3)[2];
+            String lineDescription;
+            String lineDueDate;
+            switch(lineType){
+                case "[T]":
+                    Todo oldTodo = new Todo(lineAction, Boolean.valueOf(lineIsDone));
+                    addTask(oldTodo, true);

```

Comment:
Is it really necessary to add a "true" in the method? Can consider streamlining the method.